securedrop/journalist.py

```diff
@@ -195,8 +195,13 @@ def admin_add_user():
                     flash("That username is already in use",
                           "error")
                 else:
-                    flash("An error occurred saving this user to the database",
+                    flash("An error occurred "
+                          "saving this user to the database. "
+                          "Please check the application "
+                          "logs or inform your adminstrator.",
                           "error")
+                    app.logger.error("Adding user '{}' failed: {}".format(
```

Comment:
> I don't think you should make this change here,

Do you mean it should be done elsewhere ?

I agree it would be better to log the full stack trace. Unless I missed something (very possible ;-) it is not currently done anywhere but I'd be happy to work on that if you think it is necessary in the context of this pull request. As an admin I would very much like to have additional information in the logs whenever an error happens. Instead of nothing.